--- web/system/database/drivers/mysqli/mysqli_driver.php
+++ web/system/database/drivers/mysqli/mysqli_driver.php
@@ -67,16 +67,24 @@
         * @access      private called by the base class
         * @return      resource
         */
-       function db_connect()
+    function db_connect()
        {
+               $mysqli = mysqli_init();
+               mysqli_ssl_set(
+                       $mysqli, NULL, NULL, "BaltimoreCyberTrustRoot.crt.pem", NULL, NULL
+               );
                if ($this->port != '')
                {
-                       return @mysqli_connect($this->hostname, $this->username, $this->password, $this->database, $this->port);
+                       @mysqli_real_connect(
+                               $mysqli, $this->hostname, $this->username, $this->password, $this->database, $this->port
+                       );
                } else
                {
-                       return @mysqli_connect($this->hostname, $this->username, $this->password, $this->database);
+                       @mysqli_real_connect(
+                               $mysqli, $this->hostname, $this->username, $this->password, $this->database, 3306
+                       );
                }
-
+               return $mysqli;
        }
 
        // --------------------------------------------------------------------
